Userland/Services/WebDriver/Session.cpp

```diff
@@ -57,6 +61,9 @@ ErrorOr<NonnullRefPtr<Core::LocalServer>> Session::create_server(NonnullRefPtr<S
 {
     dbgln("Listening for WebDriver connection on {}", *m_web_content_socket_path);
 
+    if (FileSystem::exists(*m_web_content_socket_path))
+        TRY(Core::System::unlink(*m_web_content_socket_path));
```

Comment:
Do we actually need the `exists()` check?
If we `unlink()` and it fails with `ENOENT`, that's okay.

Reply:
There's a TOCTOU here too, so you'd have to check for ENOENT coming from unlink anyway I think.